pkg/bpf/kepler.bpf.c

```diff
@@ -1,3 +1,5 @@
+//go:build ignore
```

Comment:
Why move this file?

Reply:
honestly - this was more of a logical move (at least to me). The file is used to generate the go implementation in `pkg/bpf/` and for that reason I moved it to reside in the package where the generated code resides